generator/altitude_generator.cpp

```diff
@@ -0,0 +1,246 @@
+#include "generator/altitude_generator.hpp"
+#include "generator/routing_generator.hpp"
+#include "generator/srtm_parser.hpp"
+
+#include "routing/routing_helpers.hpp"
+
+#include "indexer/altitude_loader.hpp"
+#include "indexer/feature.hpp"
+#include "indexer/feature_data.hpp"
+#include "indexer/feature_processor.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/file_name_utils.hpp"
+#include "coding/read_write_utils.hpp"
+#include "coding/reader.hpp"
+#include "coding/succinct_mapper.hpp"
+#include "coding/varint.hpp"
+
```

Comment:
Please, group this with other includes from coding. We usually group by top-level folder, not by sub-folders.